css/styles.css

```diff
@@ -77,3 +89,180 @@ body {
   padding: 0;
   margin: 0;
 }
+
+.works {
+  padding-inline: 24px;
+}
+
+.works-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  gap: 114px;
+  margin: 114px 0;
+  justify-content: center;
+}
+
+.card {
+  min-width: calc(327px - 16px);
+  min-height: calc(552px - 16px);
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.image-container {
+  width: 100%;
+  height: 220px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.image-container img {
+  max-width: 90%;
+}
+
+.info {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.info-title {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  margin: 0;
+}
+
+.canopy {
+  display: flex;
+  font-size: 13px;
+  font-weight: 600;
+  color: #7a869a;
+  align-items: center;
+}
+
+.canopy ul {
+  display: flex;
+  gap: 25px;
+}
+
+.extract {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.extract p {
```

Comment:
Kindly make sure you are using the same color for your paragraph. Check the difference below.
<img width="512" alt="Screenshot 2023-03-24 at 11 10 42 PM" src="https://user-images.githubusercontent.com/25887253/227629521-648633d0-116e-4eb4-9834-508caf141bc8.png">
<img width="512" alt="Screenshot 2023-03-24 at 11 10 54 PM" src="https://user-images.githubusercontent.com/25887253/227629525-acef3918-5f7f-4281-bc63-2ea276e33f11.png">